scripts/sonic-sync-mode

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
+import argparse
+from swsscommon import swsscommon
+
+def main():
+    parser=argparse.ArgumentParser(description=
+          'Configure enable or disable synchronous mode between orchagent and syncd',
+          formatter_class=argparse.RawTextHelpFormatter,
+          epilog=
+'''
+sudo needed for configuring synchronous mode
+swss restart required to apply the configuration
+Options to restart swss and apply the configuration:
+    1. config save
+       config reload -y
+    2. systemctl restart swss
+''')
+    parser.add_argument('mode', metavar='[enable|disable]', type=str,
+          help='Configure enable or disable synchronous mode')
+    args = parser.parse_args()
+
+    if args.mode == 'enable' or args.mode == 'disable':
+        config_db = swsscommon.DBConnector('CONFIG_DB', 0, False)
+        table = swsscommon.Table(config_db, 'DEVICE_METADATA')
+        fvs = swsscommon.FieldValuePairs([('synchronous_mode', args.mode)])
+        table.set('localhost', fvs)
+        print('Wrote %s synchronous mode into CONFIG_DB, swss restart required to apply the configuration'%(args.mode))
+    else:
+        print('Invalid argument: expect either enable or disable')
```

Comment:
>print [](start = 8, length = 5)

print to stderr, and return with non zero error code.

Reply:
Updated as prescribed.